js/src/sdk/models/actions.ts

```diff
@@ -180,15 +184,22 @@ export class Actions {
      */
     async list(data: GetListActionsData = {}): Promise<ActionsListResponseDTO> {
         try {
+
+            let apps = data.apps;
+            if(data?.filterByAvailableApps){
+                const integratedApps = await apiClient.appConnector.listAllConnectors();
```

Comment:
Can we add a todo note here, to create an independent api to fetch available apps instead of doing listAllConnectors flow? These API's are usually supposed to be paginated and needs to be refactored in the new major release we do.